openfisca_uk/parameters/benefit/tax_credits/working_tax_credit/elements/basic.yaml

```diff
@@ -1,12 +1,27 @@
 description: Working Tax Credit basic element
 values:
+  2002-08-01: 1_525
+  2004-04-06: 1_570
+  2005-04-06: 1_620
+  2006-04-06: 1_665
+  2007-04-06: 1_730
+  2009-04-06: 1_890
+  2010-04-06: 1_920
+  2011-04-06: 1_920
+  2012-04-06: 1_920
+  2013-04-06: 1_920
+  2014-04-06: 1_940
+  2015-04-06: 1_960
+  2016-03-16: 1_960
+  2017-04-06: 1_960
+  2018-04-06: 1_960
   2019-04-06: 1_960
-  # NB: EUROMOD reports 2_000 for 2020. This is the gov.uk value.
-  2020-04-06: 3_040
```

Comment:
Where's the legislative reference for the 3040? It's only at https://www.gov.uk/government/publications/rates-and-allowances-tax-credits-child-benefit-and-guardians-allowance/tax-credits-child-benefit-and-guardians-allowance